app/styles/_log.less

```diff
@@ -65,6 +65,10 @@
   font-family: @font-family-monospace;
   padding: 0;
   position: relative;
+  .mar-bottom-xl();
```

Comment:
It looks fine. The `log-view` bottom margin and `metrics mar-top-xl` don't combine to make too much space. 

![screen shot 2017-01-11 at 5 35 56 pm](https://cloud.githubusercontent.com/assets/1874151/21869343/264ff9c8-d825-11e6-8259-7a46a1df5ae2.png)